### DISCUSSION TITLE
Wrong directories added to PATH after installation

### DISCUSSION THREAD

**Discussion by jennings** [microsoft/winget-cli#3046] [Q&A] [answered]
On a new Windows 11 machine, I installed `fd`, `delta`, and `ripgrep` using the `winget import` command, and none of them had the correct directory added to the machine's PATH variable. Since all three have the same problem, it _seems_ like this would be a bug in winget, but I don't know enough about it to be confident. Where should I be reporting this issue, or if it's not a bug, how do I correct this behavior?

winget added these directories to PATH:

- `C:\Users\StephenJennings\AppData\Local\Microsoft\WinGet\Packages\dandavison.delta_Microsoft.Winget.Source_8wekyb3d8bbwe`
- `C:\Users\StephenJennings\AppData\Local\Microsoft\WinGet\Packages\BurntSushi.ripgrep.MSVC_Microsoft.Winget.Source_8wekyb3d8bbwe`
- `C:\Users\StephenJennings\AppData\Local\Microsoft\WinGet\Packages\sharkdp.fd_Microsoft.Winget.Source_8wekyb3d8bbwe`

However, each of them has an additional nested directory where the executable lives, so none of them can be executed from a new shell:

```
> dir -recurse C:\Users\StephenJennings\AppData\Local\Microsoft\WinGet\Packages\dandavison.delta_Microsoft.Winget.Source_8wekyb3d8bbwe

    Directory:
C:\Users\StephenJennings\AppData\Local\Microsoft\WinGet\Packages\dandavison.delta_Microsoft.Winget.Source_8wekyb3d8bbwe

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----           2/28/2023 10:10 PM                delta-0.15.1-x86_64-pc-windows-msvc

    Directory: C:\Users\StephenJennings\AppData\Local\Microsoft\WinGet\Packages\dandavison.delta_Microsoft.Winget.Sourc
e_8wekyb3d8bbwe\delta-0.15.1-x86_64-pc-windows-msvc

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---           12/3/2022  8:34 PM        6384128 delta.exe
-a---           12/3/2022  8:34 PM           1058 LICENSE
-a---           12/3/2022  8:34 PM           7046 README.md
```

```
> winget --version
v1.4.10173
```

Manifest:

```
{
  "Sources": [
    {
      "SourceDetails": {
        "Argument": "https://cdn.winget.microsoft.com/cache",
        "Identifier": "Microsoft.Winget.Source_8wekyb3d8bbwe",
        "Name": "winget",
        "Type": "Microsoft.PreIndexed.Package"
      },
      "Packages": [
        {
          "PackageIdentifier": "BurntSushi.ripgrep.MSVC"
        },
        {
          "PackageIdentifier": "dandavison.delta"
        },
        {
          "PackageIdentifier": "sharkdp.fd"
        }
      ]
    }
  ],
  "$schema": "https://aka.ms/winget-packages.schema.2.0.json",
  "WinGetVersion": "1.5.441-preview",
  "CreationDate": "2023-03-04T15:06:09.5120260-08:00"
}
```

**Comment by mdanish-kh** [comment, marked as answer]
This is a known issue and the fix is being implemented in a PR:

- https://github.com/microsoft/winget-cli/pull/3002
